### PR TITLE
Fix polyfill requirements (based on Symfony 3.3)

### PR DESCRIPTION
**Pull request by wimvds**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no?
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

The symfony/polyfill dependencies are fixed, so we can integrate Sulu with Sylius 1.1 again.

#### Why?

We were unable to combine Sylius & Sulu in projects due to dependency hell because of the symfony/polyfill requirement.

https://github.com/symfony/polyfill#usage : "When using composer you should not require the symfony/polyfill package, but the standalone ones." 

I just copied over the standalone Symfony (3.3) polyfill package requirements and it solved our dependency issues.

I don't think this will cause BC breaks ...

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -59,7 +59,11 @@
         "symfony/http-kernel": "^3.0",
         "symfony/intl": "^3.0",
         "symfony/options-resolver": "^3.0",
-        "symfony/polyfill": "^1.6",
+        "symfony/polyfill-apcu": "~1.1",
+        "symfony/polyfill-intl-icu": "~1.0",
+        "symfony/polyfill-mbstring": "~1.0",
+        "symfony/polyfill-php56": "~1.0",
```
Comment (alexander-schranz) [inline composer.json]: is this needed when we support only 7.1? /cc @wachterjohannes

File: composer.json [hunk 2]
```diff
@@ -59,7 +59,11 @@
...
+        "symfony/polyfill-php70": "~1.0",
```
Comment (alexander-schranz) [inline composer.json]: same as above

File: composer.json (hunk 1 above)
Comment (wachterjohannes) [reply composer.json]: i think the only thing which is realy needed is `symfony/polyfill-mbstring` and `symfony/polyfill-apcu` (but only in `sulu-minimal` and `sulu-standard`

Comment (wachterjohannes) [reply composer.json]: @wimvds what would you like to achieve with that? had you any problem without that?

**Comment by alexander-schranz** [comment]
at current state I think we only need: `"symfony/polyfill-mbstring"`, `"symfony/polyfill-xml"` /cc @sulu/core-team

File: composer.json (hunk 2 above)
Comment (wimvds) [reply composer.json]: No, I just copied the default polyfills from a new Symfony 3.3 project. I don't need them & I can drop them if you want.

**Comment by wimvds** [comment]
```symfony/polyfill-xml``` seems to be deprecated, should I add ```symfony/polyfill-php72``` instead (cfr. https://github.com/symfony/polyfill-xml)?

File: composer.json (hunk 2 above)
Comment (alexander-schranz) [reply composer.json]: @wimvds ok you can drop them and only add mbstring and xml as polyfill.

**Comment by alexander-schranz** [comment]
@wimvds sounds good `symfony/polyfill-php72` and `symfony/polyfill-mbstring`.

**Comment by wimvds** [comment]
@alexander-schranz the polyfills that are not needed have been removed.

**Comment by alexander-schranz** [comment]
Thank you @wimvds! We will wait for the Travis CI.

**Pull request merged**